src/components/network/network-table.js

```diff
@@ -135,6 +136,22 @@ const useStyles = makeStyles((theme) => ({
             borderBottom: '1px solid ' + theme.palette.divider,
         },
     },
+    leftFade: {
+        background:
+            'linear-gradient(to right, ' +
+            theme.palette.primary.main +
+            ', rgba(0,0,0,0) 10%)',
```

Comment:
It's up to personal preferences but with a slight border radius on the left side I reckon it looks a bit better. I tried with the following : `border-radius: 4px 0 0 4px;`

Reply:
Yes, that looks better.